axelor-account/src/main/java/com/axelor/apps/account/web/ReconcileGroupController.java

```diff
@@ -29,6 +29,23 @@
 
 public class ReconcileGroupController {
 
+  public void letter(ActionRequest request, ActionResponse response) {
+
+    ReconcileGroup reconcileGroup =
+        Beans.get(ReconcileGroupRepository.class)
+            .find(request.getContext().asType(ReconcileGroup.class).getId());
+
+    if (reconcileGroup != null) {
+      try {
+        Beans.get(ReconcileGroupService.class).letter(reconcileGroup);
```

Comment:
Need to run these two methods in a single transaction to avoid inconsistency state